src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.TimeDate/Components/SearchController.cs

```diff
@@ -56,9 +56,10 @@ internal static List<Result> ExecuteSearch(Query query, string iconTheme)
             }
 
             // Switch search type
-            if (isEmptySearchInput)
+            if (isEmptySearchInput || (!isKeywordSearch & TimeDateSettings.Instance.OnlyDateTimeNowGlobal))
```

Comment:
& -> &&

Reply:
Good catch. 🙈😅

Reply:
as soon as this is fixed, it's good to go :D